src/test/components/NetworkChart.test.js

```diff
@@ -4,14 +4,13 @@
 
 // @flow
 import * as React from 'react';
```

Comment:
I rewrote this test a bit more than the other. So you may want to scrutinize it more :)